config/README.md

```diff
@@ -233,13 +233,13 @@ Please enter issues, bugs, or suggestions as
 Copyright (c) Microsoft Corporation. All rights reserved.
 Licensed under the [MIT](LICENSE) License.
 
-[build-badge]: https://img.shields.io/travis/Azure/pcs-config-java.svg
-[build-url]: https://travis-ci.org/Azure/pcs-config-java
+[build-badge]: https://solutionaccelerators.visualstudio.com/RemoteMonitoring/_apis/build/status/Consolidated%20Repo%20-%20Java 
+[build-url]: https://solutionaccelerators.visualstudio.com/RemoteMonitoring/_build/latest?definitionId=25
```

Comment:
I would also say lets just remove the build badge completely from the services list because it takes exactly to the same build definition as top-level build badge. When we have badge available for individual services when we can run partial builds then we can add it back